Sources/Purchasing/Purchases/Purchases.swift

```diff
@@ -204,9 +206,26 @@ public typealias StartPurchaseBlock = (@escaping PurchaseCompletedBlock) -> Void
         }
     }
 
+    /**
+     * Set a custom handler for redirecting errors to your own logging system.
+     * This does nothing by default, but it can be overriden to detect and handle
+     * any error occuring in the SDK.
+     *
+     * #### Related Symbols
+     * - ``logHandler``
+     * - ``verboseLogHandler``
+     * - ``logLevel``
+     */
+    @objc public static var errorHandler: ErrorHandler {
```

Comment:
my only question is: without reading the docstring, would it be immediately obvious to a developer that this doesn't actually handle the errors, but rather error _logs_?
I.e.: I might see this and assume that we do a mega try-catch where everything that's caught is handled by this, so that I have a single place to show a "something went wrong" alert or toast. 
Rather than realize that this is actually about errors. 
Maybe errorLogHandler or something might help?

Reply:
Great point 👍🏻  I'll change that.